Cython/Compiler/Pythran.py

```diff
@@ -154,7 +163,7 @@ def to_pythran(op, ptype=None):
     if is_type(op_type, ["is_pythran_expr", "is_numeric", "is_float", "is_complex"]):
         return op.result()
     if op.is_none:
-        return "pythonic::__builtin__::None"
+        return "pythonic::{}::None".format(pythran_builtins)
```

Comment:
Since Cython uses `%` formatting everywhere for code generation, please also stick to it here.

Reply:
Done!